packages/plugin-ext/src/hosted/node/hosted-plugin-localization-service.ts

```diff
@@ -62,26 +65,77 @@ export class HostedPluginLocalizationService implements BackendApplicationContri
             .then(() => this._ready.resolve());
     }
 
-    deployLocalizations(plugin: DeployedPlugin): void {
+    async deployLocalizations(plugin: DeployedPlugin): Promise<void> {
+        const disposable = new DisposableCollection();
         if (plugin.contributes?.localizations) {
             const localizations = buildLocalizations(plugin.contributes.localizations);
```

Comment:
So these are translations contributed inside the package.json?

Reply:
Ah...we read package.nls.json to i18n the strings inside package.json here, right? This would be worth a comment.